.gitignore

```diff
@@ -1 +1,2 @@
 /bazel-*
+.vscode
```

Comment:
Exclusions like this should be placed in `~/.gitignore` instead of adding all/random IDEs to project's `.gitignore`.

Reply:
Ack - some projects I work on(envoy, istio) keep them so force of habit. Will remove